bladeui/css/blade.scss

```diff
@@ -1,24 +1,30 @@
 @import 'blade/common.scss';
-@import 'blade/getStarted.scss';
-@import 'blade/termsAndConditions.scss';
-@import 'blade/secretPhrase.scss';
-@import 'blade/confirmSecretPhrase.scss';
-@import 'blade/verifyKyc.scss';
-@import 'blade/createPassword.scss';
-@import 'blade/setExternalWallet.scss';
-@import 'blade/registrationCompleted.scss';
-@import 'blade/about.scss';
-@import 'blade/recoverPhrase.scss';
-@import 'blade/recoverPassword.scss';
-@import 'blade/recoverCompleted.scss';
-@import 'blade/refferals.scss';
 @import 'blade/statsRepresentation.scss';
-@import 'blade/refferalsList.scss';
-@import 'blade/referralsForm.scss';
 @import 'blade/loader.scss';
-@import 'blade/manualTransfer.scss';
 @import 'blade/infiniteScrollLoader.scss';
-@import 'blade/referralCode.scss';
-@import 'blade/transfersList.scss';
-@import 'blade/transfers.scss';
-@import 'blade/profile.scss';
+
+@import 'settings/about.scss';
+@import 'settings/transfers.scss';
+@import 'settings/profile.scss';
+
+@import 'transfers/manualTransfer.scss';
+@import 'transfers/transfersList.scss';
+
+@import 'pagesForUnregisteredUsers/createPassword.scss';
```

Comment:
this page uses in registration

Reply:
this section name is very strange, please divide this files to appropriate folders
You can leave here only getStarted page

Reply:
this naming is after your constant name in baseClass (PAGES_ALLOWED_FOR_UNREGISTERED)